session/session.go

```diff
@@ -1385,6 +1386,13 @@ func (s *session) ParseWithParams(ctx context.Context, sql string, args ...inter
 	for _, warn := range warns {
 		s.sessionVars.StmtCtx.AppendWarning(util.SyntaxWarn(warn))
 	}
+	if config.TopSQLEnabled() {
+		normalized, digest := parser.NormalizeDigest(sql)
+		if digest != nil {
+			// Fixme: reset/clean the label when sql execute finish.
+			tracecpu.SetGoroutineLabelsWithSQL(ctx, normalized, digest.String())
```

Comment:
Should we exclude sql text from the label?

Reply:
Ditto. The SQL test use to register into the collector instead of being a label.